### PR TITLE
hubstaff: 1.5.2-bead991b -> 1.5.19-9e79d1da

### PR DESCRIPTION
**Pull request by srghma**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by srghma** [comment]
Could someone help with this, I have an error

```
 ✘  ~/projects/nixpkgs   hubstaff_update  HubstaffClient --help
2021-06-14 20:23:00.046245 [DEBUG] [0x27dfe70] ui_style_adwaita.cpp:439  DEfinineg adwaita
2021-06-14 20:23:00.046427 [DEBUG] [0x27dfe70] fltk.cpp:265  Define scheme: adwaita
2021-06-14 20:23:00.046476 [DEBUG] [0x27dfe70] fltk.cpp:277  Define scheme: adwaita first box: 64
2021-06-14 20:23:00.063220 [DEBUG] [0x27dfe70] fltk.cpp:265  Define scheme: cairo
2021-06-14 20:23:00.063314 [DEBUG] [0x27dfe70] fltk.cpp:277  Define scheme: cairo first box: 67
2021-06-14 20:23:00.063355 [DEBUG] [0x27dfe70] fltk_cairo.cpp:565  cairo_box_start : 67
2021-06-14 20:23:00.063515 [DEBUG] [0x27dfe70] Log.cpp:188  Echoing to stdout : 1
2021-06-14 20:23:00.063558 [DEBUG] [0x27dfe70] main.cpp:23   Hubstaff client started
2021-06-14 20:23:00.065739 [DEBUG] [0x27dfe70] LinuxXGrabber.cpp:81   Enhanced Window Manager is present, name = i3
2021-06-14 20:23:00.066147 [DEBUG] [0x27dfe70] LinuxXGrabber.cpp:326  id: kitty name: kitty binary: /nix/store/2lwgcgs57d83ccf49441z2jdk3y5vad4-kitty-0.20.3/bin/.kitty-wrapped
2021-06-14 20:23:00.066359 [INFO] [0x27dfe70] Log.cpp:277  ------------------------------------------
2021-06-14 20:23:00.066444 [INFO] [0x27dfe70] Log.cpp:278  Log opened.
2021-06-14 20:23:00.066545 [INFO] [0x27dfe70] Log.cpp:283  Local Time : 2021-06-14 20:23:00
2021-06-14 20:23:00.066623 [INFO] [0x27dfe70] Log.cpp:284  UTC Time   : 2021-06-14 17:23:00
2021-06-14 20:23:00.066710 [INFO] [0x27dfe70] Log.cpp:285  TZ Offset  : 03:00:00
2021-06-14 20:23:00.066768 [INFO] [0x27dfe70] Log.cpp:286  ------------------------------------------
2021-06-14 20:23:00.066822 [INFO] [0x27dfe70] Platform.cpp:104  Last shutdown was clean
2021-06-14 20:23:00.066875 [INFO] [0x27dfe70] Platform.cpp:106  Client Version:  1.5.19-9e79d1da
2021-06-14 20:23:00.066928 [INFO] [0x27dfe70] Platform.cpp:107  Client Platform: Linux x86_64
2021-06-14 20:23:00.070431 [INFO] [0x27dfe70] Platform.cpp:108  Client OS:       Linux machine 5.10.37 #1-NixOS SMP Fri May 14 07:50:46 UTC 2021 x86_64 GNU/Linux
2021-06-14 20:23:00.070564 [INFO] [0x27dfe70] Platform.cpp:109  Client MEM:
2021-06-14 20:23:00.070633 [INFO] [0x27dfe70] Platform.cpp:110  Client Data:     /home/srghma/.local/share/HubStaff
2021-06-14 20:23:00.070694 [INFO] [0x27dfe70] Platform.cpp:111  ------------------------------------------
2021-06-14 20:23:00.072883 [DEBUG] [0x27dfe70] Configuration.cpp:108  Loaded Config
account
{
    email melgaardbjorn@gmail.com
    user_id 232318
    profile a292df8d98b1708be1e1f85804ed0bc877278379
    api_token uaHUQ5Ij3gIeB2vCXff4ndmg8cwakgwtO0FevZA9KpQ
}
client
{
    widget
    {
        visible false
        x 521
        y 202
        w 1228
        h 648
        projects
        {
            w 332
        }
        details
        {
            h 207
        }
        tasks
        {
            created_at 120
            due_at 120
            remote_id 120
            summary 246
            updated_at 120
        }
        expanded true
        tasksExpanded true
    }
    client_id u0ztbDNPUxLjrMzfiU8gBiu9G8BsLVKlapLGXlcwPIs
    preferences
    {
        autostart false
        main_window_close_action 1
        always_on_top true
        show_screenshot_notification true
        taskbar_behavior 1
    }
    onboard_version 1
    device_id fa7fe8e7-916b-441a-ad50-aae840fe554a
}
screen
{
    max_width 1280
    max_height 1280
    quality 90
}
activity
{
    report_interval 600
    idle_timeout 300
    resume_threshold
    {
        keep 06:00:00
        discard 06:00:00
    }
}

2021-06-14 20:23:00.098711 [DEBUG] [0x27dfe70] fltk.cpp:168  FLTK_STATE init
Fontconfig warning: "/etc/fonts/2.11/fonts.conf", line 5: unknown element "description"
Fontconfig error: "/etc/fonts/2.11/fonts.conf", line 5: invalid attribute 'domain'
Fontconfig warning: "/etc/fonts/conf.d/10-hinting-slight.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/10-scale-bitmap-fonts.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/20-unhint-small-vera.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/30-metric-aliases.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/40-nonlatin.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/45-generic.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/45-latin.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/49-sansserif.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/50-user.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/51-local.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/60-generic.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/60-latin.conf", line 4: unknown element "description"
Fontconfig warning: "/etc/fonts/conf.d/65-nonlatin.conf", line 4: unknown element "description"
Fontconfig error: Cannot load config file from /etc/fonts/2.11/fonts.conf
Unable to find fonts. Check your FontConfig configuration.
```

**Comment by srghma** [comment]
the error didnt appear until updated to nixos 21

**Comment by srghma** [comment]
Question: how can I make the https://github.com/NixOS/nixpkgs/blob/f23d953cbebe59f716f8b8df0c583b904dd6e2a1/pkgs/applications/misc/hubstaff/update.sh to be executed automatically

**Comment by SuperSandro2000** [comment]
> Question: how can I make the [`f23d953`/pkgs/applications/misc/hubstaff/update.sh](https://github.com/NixOS/nixpkgs/blob/f23d953cbebe59f716f8b8df0c583b904dd6e2a1/pkgs/applications/misc/hubstaff/update.sh) to be executed automatically by bot

See https://github.com/NixOS/nixpkgs/blob/35832045ceab6954f4a3116d77f2a06c72e2424c/pkgs/tools/typesetting/asciidoctor/default.nix#L26-L28

**Comment by srghma** [comment]
@SuperSandro2000 done

File: pkgs/applications/misc/hubstaff/revision.json [hunk 1]
```diff
@@ -1,5 +1,2 @@
 {
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/hubstaff/revision.json:1]:
```suggestion
```

File: pkgs/applications/misc/hubstaff/revision.json [hunk 2]
```diff
@@ -1,5 +1,2 @@
...
-  "url": "https://hubstaff-production.s3.amazonaws.com/downloads/HubstaffClient/Builds/Release/1.5.15-28673d1c/Hubstaff-1.5.15-28673d1c.sh",
-  "version": "1.5.15-28673d1c",
-  "sha256": "0ging41l3a3sdj6bggh913h71gqmb8l3rplp4civpgclnfzv7fg2"
 }
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/hubstaff/revision.json:2]:
```suggestion
```

File: pkgs/applications/misc/hubstaff/revision.json [hunk 3]
```diff
@@ -1,5 +1 @@
-{
-  "url": "https://hubstaff-production.s3.amazonaws.com/downloads/HubstaffClient/Builds/Release/1.5.15-28673d1c/Hubstaff-1.5.15-28673d1c.sh",
-  "version": "1.5.15-28673d1c",
-  "sha256": "0ging41l3a3sdj6bggh913h71gqmb8l3rplp4civpgclnfzv7fg2"
-}
+
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/hubstaff/revision.json:1]:
```suggestion
```

**Pull request merged**